node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
All of the files in the `node_modules/` directory should be ignored. You'll need to, locally, `git rm node_modules/.yarn-integrity`, commit it (I think the best type might be "build") and continue to get rid of this file. If it's not auto-generated you should copy it somewhere else first, then after removing it from git you can return it here, if it's necessary, though it's probably auto-generated. My version of this file is very different